.php-cs-fixer.dist.php

```diff
@@ -28,8 +30,20 @@
         '@PHPUnit75Migration:risky' => true,
         '@Symfony' => true,
         '@Symfony:risky' => true,
+        'protected_to_private' => false,
+        'no_superfluous_phpdoc_tags' => [
+            'remove_inheritdoc' => true,
+            'allow_unused_params' => true, // for future-ready params, to be replaced with https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/7377
+        ],
         'header_comment' => ['header' => $fileHeaderComment],
-        'trailing_comma_in_multiline' => ['elements' => ['arrays', 'match', 'parameters']],
+        // TODO: Remove once the "compiler_optimized" set includes "sprintf"
+        'native_function_invocation' => ['include' => ['@compiler_optimized', 'sprintf'], 'scope' => 'namespaced', 'strict' => true],
+        'nullable_type_declaration' => true,
+        'nullable_type_declaration_for_default_null_value' => true,
```

Comment:
Not needed anymore... where do you take your file ? 😅 

https://github.com/symfony/symfony/blob/7.3/.php-cs-fixer.dist.php

Reply:
Mmmmh...... mmmmmmmmmh.... 🤦🏻  definitly not from the last working branch, thanks, I will do the changes ASAP  😅